presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/EnvironmentUp.java

```diff
@@ -68,11 +68,13 @@ public void run()
             implements Runnable
     {
         private final SelectedEnvironmentProvider selectedEnvironmentProvider;
+        private final Boolean withoutPrestoMaster;
```

Comment:
Any reason not to boolean ?

```suggestion
        private final boolean withoutPrestoMaster;
```